### PR TITLE
New console SSO sub section

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-aura-249.surge.sh

**Comment by MarkWoulfeNeo4j** [comment]
[Document Console SSO](https://trello.com/c/L9cCwZrC/206-document-console-sso)

File: modules/ROOT/pages/platform/security.adoc [hunk 1]
```diff
@@ -179,28 +179,43 @@ Please see the https://learn.microsoft.com/en-us/azure/private-link/rbac-permiss
...
-Aura supports Authorization Code Flow with PKCE to ensure best practice security.
+The following OpenID Connect (OIDC) certified Identity Providers (IdPs) are currently supported for instance-level Authentication:
+
+* Microsoft Azure Active Directory (AAD)
+* Okta
+* Keycloak
+* Google Authentication
 
-To add SSO for Browser and Bloom to your AuraDB Enterprise instances, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket] including the following information:
+To add SSO for Workspace, Bloom and Browser to your Aura Enterprise instances, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket] including the following information:
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/security.adoc:214]: Oxford comma here? We do recommend it in our style guide

File: modules/ROOT/pages/platform/security.adoc [hunk 2]
```diff
@@ -179,28 +179,43 @@ Please see the https://learn.microsoft.com/en-us/azure/private-link/rbac-permiss
...
+
+. The _Tenant ID_ of the tenant(s) you want to use SSO. See xref:platform/user-management.adoc#_tenants[Tenants] for more information on how to find your __Tenant ID__.
+. The name of your IdP.
+
+=== Instance SSO
+
+Instance SSO allows you to directly map groups of users (as defined in your IdP) to DBMS RBAC roles when launching Workspace, Bloom and Browser clients from an Aura instance.
 
-Aura supports Authorization Code Flow with PKCE to ensure best practice security.
+The following OpenID Connect (OIDC) certified Identity Providers (IdPs) are currently supported for instance-level Authentication:
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/security.adoc:207]: The IdP acronym is redefined here, can we skip it here or move the definition before both sections?

File: modules/ROOT/pages/platform/security.adoc [hunk 3]
```diff
@@ -170,7 +170,7 @@ image::azure_privatelink_03_browser_bloom_over_vpn.png[title="Accessing Browser
 
 ==== Enabling private endpoints
 
-To enable private endpoints using Azure Private Link, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket], and we’ll be in touch.
+To enable private endpoints using Azure Private Link, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket], and we'll be in touch.
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/security.adoc:173]:
```suggestion
To enable private endpoints using Azure Private Link, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket].
```

I think we can skip this bit

File: modules/ROOT/pages/platform/security.adoc [hunk 4]
```diff
@@ -179,28 +179,43 @@ Please see the https://learn.microsoft.com/en-us/azure/private-link/rbac-permiss
...
+* Okta
+* Keycloak
+* Google Authentication
 
-To add SSO for Browser and Bloom to your AuraDB Enterprise instances, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket] including the following information:
+To add SSO for Workspace, Bloom and Browser to your Aura Enterprise instances, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket] including the following information:
 
 . The *Connection URI* of the instance(s) you want to use SSO.
-. Whether or not you want Browser, Bloom, or both enabled.
+. Whether or not you want Workspace, Bloom, Browser or a combination of them enabled.
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/security.adoc:217]:
```suggestion
. Whether or not you want Workspace, Bloom, Browser, or a combination of them enabled.
```

Not sure about comma here, but I guess same reasoning as previous comment

File: modules/ROOT/pages/platform/security.adoc (hunk 1 above)
Comment (MarkWoulfeNeo4j) [reply modules/ROOT/pages/platform/security.adoc:214]:
```suggestion
To add SSO for Workspace, Bloom, and Browser to your Aura Enterprise instances, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket] including the following information:
```

File: modules/ROOT/pages/platform/security.adoc (hunk 2 above)
Comment (MarkWoulfeNeo4j) [reply modules/ROOT/pages/platform/security.adoc:207]:
```suggestion
The following OIDC certified IdPs are currently supported for instance-level Authentication:
```

**Review by nvitucci — APPROVED** [review]

**Pull request merged**